chia/full_node/full_node.py

```diff
@@ -97,54 +97,55 @@ class FullNode:
     sync_store: Any
     coin_store: CoinStore
     mempool_manager: MempoolManager
-    _sync_task: Optional[asyncio.Task]
-    _init_weight_proof: Optional[asyncio.Task] = None
+    _sync_task: Optional[asyncio.Task[None]]
+    _init_weight_proof: Optional[asyncio.Task[None]] = None
     blockchain: Blockchain
-    config: Dict
+    config: Dict[str, Any]
     server: Any
     log: logging.Logger
     constants: ConsensusConstants
     _shut_down: bool
     root_path: Path
-    state_changed_callback: Optional[Callable]
+    state_changed_callback: Optional[Callable[..., Any]]
```

Comment:
Do we really not have a return type for this?  And how do we call it if it can have any number of parameters?